packages/siop-oid4vp/lib/authorization-request/AuthorizationRequest.ts

```diff
@@ -256,7 +270,11 @@ export class AuthorizationRequest {
   }
 
   public async mergedPayloads(): Promise<RequestObjectPayload> {
-    return { ...this.payload, ...(this.requestObject && (await this.requestObject.getPayload())) }
+    const  requestObjectPayload = { ...this.payload, ...(this.requestObject && (await this.requestObject.getPayload())) }
+    if (requestObjectPayload.scope && typeof requestObjectPayload.scope !== 'string') { //  test mattr.launchpad.spec.ts does not supply a scope value
```

Comment:
The situation I found was 
![image](https://github.com/user-attachments/assets/8b59149e-7686-4161-ad4c-e0afc69c4cd8)

Not sure why Sadjad placed it precisely there, maybe because mergedPayloads() is called in many nasty/embedded places.
But the field should be there
> scope: string // REQUIRED. As specified in Section 3.1.2 of [OpenID.Core].

anyway the mattr.launchpad.spec.ts does not supply a requestObject so we also won't have a scope either.
so again these tests should either be upgraded or disabled imo

Reply:
Well something must be off with the mattr test or the way we treat request objects then. The test has been there for a very long time and always passed.

The lib should be able to work without a request object (so in plain oauth2 mode)

Reply:
Also the string assertion only applies of there is a scope to begin with. If not then the error will never be thrown